### DISCUSSION TITLE
how to use plugin-syntax-import-meta

### DISCUSSION THREAD

**Discussion by a6965921** [webpack/webpack#15826] [Q&A]
![image](https://user-images.githubusercontent.com/1198840/168414136-8e095876-a7c6-4709-bc4b-bc3ff2e97365.png)

**Comment by alexander-akait** [comment]
What is webpack-dev-server version?

**Reply by a6965921** [reply]
3.12.0

**Reply by vankop** [reply]
you trying to use babel plugin in webpack config... maybe you want to use it in babelrc?
